data/json/mutations.json

```diff
@@ -630,7 +630,7 @@
         "points" : -2,
         "description" : "You have problems with eating meat.  It's possible for you to eat it, but you will suffer morale penalties due to nausea.",
         "starting_trait" : true,
-        "//" : "Intolerances are presumed a psychological issue unless stated otherwise, and cancelled by digestive mutations, which force the issue.",
+        "note" : "Intolerances are presumed a psychological issue unless stated otherwise, and cancelled by digestive mutations, which force the issue.",
```

Comment:
The "//" seems to be better, it's used (nearly) everywhere else and our JSON parser has special code to ignore repeated members of that name.